apps/home/blocks.py

```diff
@@ -83,7 +83,9 @@ class Meta:
 class ParagraphBlock(blocks.StructBlock):
     body = blocks.RichTextBlock(
         required=True,
-        help_text=HELPTEXT_RICHTEXT_A11Y
+        help_text=HELPTEXT_RICHTEXT_A11Y,
+        features=['bold', 'italic', 'h2', 'h3', 'ol', 'ul', 'hr', 'link',
+                  'document-link', 'image']
```

Comment:
this comes from 
https://docs.wagtail.org/en/stable/reference/streamfield/blocks.html#wagtail.blocks.RichTextBlock
links to following even though it is referencing RichTextField not Block
https://docs.wagtail.org/en/stable/advanced_topics/customisation/page_editing_interface.html#rich-text-features